src/pytorch_lightning/loops/optimization/manual_loop.py

```diff
@@ -81,17 +81,16 @@ def __init__(self) -> None:
         # `OptimizationProgress`
         self.optim_step_progress = Progress.from_defaults(ReadyCompletedTracker)
 
-        self._done: bool = False
```

Comment:
In many loops we had to rely on instance attributes to share information between methods to conform to the base `run` interface.

This is no longer required so we can look for and remove these quirks. This is one example